core/store/src/columns.rs

```diff
@@ -247,9 +247,10 @@ pub enum DBCol {
     /// - *Rows*: height (u64)
     /// - *Column type*: Vec<HeaderHashes (CryptoHash)>
     HeaderHashesByHeight,
-    /// State changes made by a chunk, used for splitting states
+    /// State changes made by a chunk, used for resharding. Historically
+    /// resharding was also called State Splitting since the name.
     /// - *Rows*: BlockShardId (BlockHash || ShardId) - 40 bytes
-    /// - *Column type*: StateChangesForSplitStates
+    /// - *Column type*: StateChangesForResharding
     StateChangesForSplitStates,
```

Comment:
Wondering whether it makes sense to do something like this?
```    
    #[strum(serialize = "StateChangesForSplitStates")]
    StateChangesForResharding,
```

Reply:
This way the serialization for RocksDB still uses `StateChangesForSplitStates` but in code we use `DbCol:: StateChangesForResharding`

Reply:
Just found this in our codebase `DBCol::StateChangesForSplitStates => "col49",`
Seems like we name StateChangesForSplitStates as "col49" in RocksDB.

Reply:
That looks great but let me follow up separately as this would require some proper testing. I'll add a TODO and an item in the tracking issue.